README.md

```diff
@@ -70,6 +70,7 @@ packages/
   > Handles processing of post-office counter sales
 - [fulfilment-job](packages/fulfilment-job/README.md)
   > Handles processing of licence fulfilment requirements
+- [recurring-payments-job](packages/recurring-payments-job/README.md)
```

Comment:
Can we also have a line of description of what the job does, like the others do above?